crates/biome_js_analyze/src/lint/nursery/no_bind.rs

```diff
@@ -0,0 +1,67 @@
+use biome_analyze::{context::RuleContext, declare_lint_rule, Ast, Rule, RuleDiagnostic};
+use biome_console::markup;
+use biome_js_syntax::JsIdentifierBinding;
+use biome_rowan::AstNode;
+
+declare_lint_rule! {
+    /// Succinct description of the rule.
+    ///
+    /// Put context and details about the rule.
+    /// As a starting point, you can take the description of the corresponding _ESLint_ rule (if any).
+    ///
+    /// Try to stay consistent with the descriptions of implemented rules.
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
+    ///
+    /// ```js,expect_diagnostic
+    /// var a = 1;
+    /// a = 2;
+    /// ```
+    ///
+    /// ### Valid
+    ///
+    /// ```js
+    /// // var a = 1;
+    /// ```
+    ///
+    pub NoBind {
+        version: "next",
+        name: "noBind",
```

Comment:
I feel like we can come up with a better name than `noBind`. At the very least, it should mention "Jsx" since this rule should only apply to Jsx.

Reply:
Oh, I see😅.
However, I've noticed that some existing JSX rules include "jsx" in their names while others do not. Maybe need to standardize "Jsx" naming convention.